### PR TITLE
Cocoapod and swift 3 support

### PR DESCRIPTION
**Pull request by jakubknejzlik**

### CODE REVIEW

File: Example/Pesticide/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    func applicationDidBecomeActive(_ application: UIApplication) {
+        // Restart any tasks that were paused (or not yet started) while the application was inactive. If the application was previously in the background, optionally refresh the user interface.
+    }
+
+    func applicationWillTerminate(_ application: UIApplication) {
+        // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
+    }
+
```
Comment (jakubpetrik) [inline Example/Pesticide/AppDelegate.swift]: Can you please remove those empty stubs?

File: LICENSE [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2014 Pesticide
+Copyright (c) 2016 Jakub Knejzlik <jakub.knejzlik@gmail.com>
```
Comment (jakubpetrik) [inline LICENSE]: We should probably use Inloop copyright. :)

File: Pesticide.podspec [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+  s.description      = <<-DESC
+TODO: Add long description of the pod here.
+                       DESC
+
+  s.homepage         = 'https://github.com/inloop/Pesticide'
+  # s.screenshots     = 'www.example.com/screenshots_1', 'www.example.com/screenshots_2'
+  s.license          = { :type => 'MIT', :file => 'LICENSE' }
+  s.author           = { 'Jakub Knejzlik' => 'jakub.knejzlik@gmail.com' }
+  s.source           = { :git => 'https://github.com/inloop/Pesticide.git', :tag => s.version.to_s }
```
Comment (jakubpetrik) [inline Pesticide.podspec:28]: Same here. :)

File: Pesticide/Classes/DebugTableController.swift [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+        }
+        self.readCurrentLog()
+        self.tableView.tableHeaderView = self.consoleView
+
+        keyboardDoneButtonView.barStyle = UIBarStyle.default
+        keyboardDoneButtonView.isTranslucent = false
+        keyboardDoneButtonView.tintColor = nil
+        keyboardDoneButtonView.sizeToFit()
+
+        let doneButton = UIBarButtonItem(title: "Done", style: UIBarButtonItemStyle.done, target: self, action: #selector(DebugTableController.pickerViewDone))
```
Comment (jakubpetrik) [inline Pesticide/Classes/DebugTableController.swift:46]: Please remove all redundant `self` references.

File: Pesticide/Classes/DebugView.swift [hunk 5]
```diff
@@ -12,14 +12,15 @@ class DebugView : UIView {
 
     override init(frame: CGRect) {
         super.init(frame: frame)
+        initInit()
```
Comment (jakubpetrik) [inline Pesticide/Classes/DebugView.swift]: :D wth?! Rename please.

File: Pesticide/Classes/Pesticide.swift [hunk 6]
```diff
@@ -6,20 +6,22 @@
...
+    case button
+    case `switch`
+    case slider
+    case dropdown
+    case label
+    case header
 }
 
-public class Pesticide {
+open class Pesticide {
```
Comment (jakubpetrik) [inline Pesticide/Classes/Pesticide.swift]: Do we really need this to be open? Why not `public final`?

File: Pesticide/Classes/DebugTableController.swift (hunk 4 above)
Comment (jakubknejzlik) [reply Pesticide/Classes/DebugTableController.swift:46]: This comment is not removed by recet changes, but I went through all files and removed self reference where it wasn't necessary.

**Pull request merged**